packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -1130,12 +1141,12 @@ export default class Meetings extends WebexPlugin {
 
       if (enableUnifiedMeetings && !isMeetingActive && useRandomDelayForInfo && waitingTime > 0) {
         meeting.fetchMeetingInfoTimeoutId = setTimeout(
-          () => meeting.fetchMeetingInfo({}),
+          () => meeting.fetchMeetingInfo({extraParams: infoExtraParams}),
           waitingTime
         );
         meeting.parseMeetingInfo(undefined, destination);
       } else {
-        await meeting.fetchMeetingInfo({});
+        await meeting.fetchMeetingInfo({extraParams: infoExtraParams});
```

Comment:
question: do we need to provide these extra params in other places when meeting info is fetched, for example in Meeting.verifyPassword?

Reply:
The use case I'm currently trying to solve was for meeting create. I can add this to `verifyPassword` also, but I couldn't think of a use case right now for it (the create case needs the extra params to provide `mtid` / `joinTXId` which are instead of a password / captcha).

If you'd like this added let me know and I'll make the change.

Reply:
ok, looks like it is called only from `verifyPassword()` and from `PersonalMeetingRoom.get()` which I guess both don't actually need the mtid so it's OK to keep it as it is